testing/spectest/minimal/electra/fork_transition/BUILD.bazel

```
@@ -0,0 +1,7 @@
+load("@prysm//tools/go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    srcs = ["transition_test.go"],
+    deps = ["//testing/spectest/shared/electra/fork:go_default_library"],
+)
```

```suggestion
go_test(
    name = "go_default_test",
    srcs = ["transition_test.go"],
    deps = ["//testing/spectest/shared/electra/fork:go_default_library"],
    network = "minimal",
    data = glob(["*.yaml"]) + [
        "@consensus_spec_tests_minimal//:test_data",
    ],
)
```

Run gazelle too, i dont think i put the fields in the right order